modules/aioseop_sitemap.php

```diff
@@ -2968,6 +3220,27 @@ function get_prio_from_posts( $posts, $prio_override = false, $freq_override = f
 
 
 					$pr_info['image:image'] = $is_single ? $this->get_images_from_post( $post ) : null;
+
+					/**
```

Comment:
The arrays had quite a bit of documentation missing.

```
/**
 * Filter to update the data for a particular <url> of the sitemap.
 *
 * @since ?
 *
 * @param array $pr_info {
 *     An array of arrays, where each array corresponds to the <url> element of the sitemap.
 *
 *     @type string $changefreq Amount of time to update.
 *     @type array  $image:image {
 *          Image data.
 *
 *          TODO Document missing keys.
 *     }
 *     @type string $lastmod    Last time the post has been modified.
 *     @type string $priority   Priority of the post.
 *     @type array  $rss {
 *         @type string $description Post description.
 *         @type string $post_type   What post type the object belongs to.
 *         @type string $pubDate     Publish date.
 *         @type int    $timestamp   Epoch timestamp.
 *         @type string $title       Post title.
 *     }
 * }
 * @param WP_Post $post {
 *     The post object.
 *
 *     @see WP_Post
 *     @link https://codex.wordpress.org/Class_Reference/WP_Post
 * }
 * @param array $args {
 *     Key/value array of arguments that signify whether.
 *
 *     @type boolean $fre_override  Priority that needs to be overridden.
 *     @type string  $linkfunc      Custom function used to determine the permalink to the post.
 *     @type boolean $prio_override Frequency that needs to be overridden.
 * }
 */
```